app/inspect/index.tsx

```diff
@@ -263,6 +246,7 @@ const Index = () => {
           frameProcessor={frameProcessor}
           video={true}
           audio={true}
+          style={{ width: '25%', height: '25%' }}
```

Comment:
Please remember to remove the camera preview styling (`style={{ width: '25%', height: '25%' }}`) before finalizing the feature, as we don't want to display the camera preview during inspection.

Reply:
yeah its also for testing purposes during development
in final version it will be removed.